boilerplates/ds-ml-service/files/README.md

```diff
@@ -1,231 +1 @@
-      DSI_SSH_HTTPS_PROXY: https://proxy
-``` 
-
-`docker-compose -f docker-compose.yml -f docker-compose.ssh.yml up`  
-
-The training pod starts an asynchronous training task. Only one 
-training task can run at a time. 
-
-## Endoints ##
-
-### Training Endpoint ###
-* `/` : Return all information about the training service 
-* `/start` : Starts the training. 
-* `/finished` : Checks if the current traning task is finished
-* `/getmodel` : Download the latest trained model
-
-### Prediction Endpoint ###
-* `/predict` : Return all information about the training service
-    * payload: Should be a json containing the data necessary for prediciton. The payload is not pre defined, but it is defined by the trainined model 
- 
-
-
-There is not need for any kind of payload in all endpoints. 
-
-### Environment Variables for training ###
-
-| Environment Variable | Description | Allowed Values |
-|------|-------|---|
-| DSI_DEBUG_MODE | Enables debug mode | true, 1 our yes for debug mode, otherwise debug is disasbled|
-| DSI_EXECUTE_ON | Where the train should be executed | LOCAL, SSH |
-| DSI_TRAINING_SERVICE_USERNAME | Username to be set as default username for accessing the services | string, required |
-| DSI_TRAINING_SERVICE_PASSWORD | Password to be set as default password for accessing the services | string, required |
-| |Following variables are applicable if `DSI_EXECUTE_ON=SSH`| |
-| DSI_SSH_HOST | SSH host name where train should be executed (Only applicable if DSI_EXECUTE_ON=SSH) | host names or ip addresses |
-| DSI_SSH_PORT | SSH host port where train should be executed (Only applicable if DSI_EXECUTE_ON=SSH) | port numbers (Default: 22) |
-| DSI_SSH_USERNAME | SSH username for remote execution  | string\ |
-| DSI_SSH_PASSWORD | SSH password for remote execution  | string |
-| DSI_SSH_HTTP_PROXY | HTTP proxy url for remote execution. This is needed if the remote machine needs the proxy for download packages and resources  | string |
-| DSI_SSH_HTTPS_PROXY | HTTPS proxy url for remote execution. This is needed if the remote machine needs the proxy for download packages and resources  | string |
-
-### Environment Variables for prediction ###
-
-| Environment Variable | Description | Allowed Values |
-|------|-------|---|
-| DSI_DEBUG_MODE | Enables debug mode | true, 1 our yes for debug mode, otherwise debug is disasbled|
-| DSI_TRAINING_BASE_URL | The base url where the prediction service should get the model from | url (e.g. https://training.OpenShift.svc |
-| DSI_TRAINING_SERVICE_USERNAME | Username of the training service | string, required |
-| DSI_TRAINING_SERVICE_PASSWORD | Password of the training service | string, required |
-| DSI_PREDICTION_SERVICE_USERNAME | Username to be set as default username for accessing the service | string, required |
-| DSI_PREDICTION_SERVICE_PASSWORD | Password to be set as default password for accessing the service | string, required |
+# Data Science Industrialization Service for ...
```

Comment:
what if we mv/cp the readme to files through build.sh script when provisioning the component so the users can have the whole readme too in their repos at the beginning?